pages/implementation/start-using-inspector-with-avo-functions.mdx

```diff
@@ -105,6 +115,47 @@ Avo.initAvo({ env: Avo.AvoEnv.Dev, inspector: inspector }, ...);
 
 </details>
 
+---
+
+<details>
+<summary><strong>Node</strong></summary>
+<p/>
+To enable Avo Functions to report to Avo Inspector automatically you would need to provide an instance of Avo Inspector, <a target="\_blank" rel="noopener noreferrer" href="https://github.com/avohq/node-avo-inspector">check the quick start guide for how to create one</a>, when initializing Avo and use Avo version >= `71.34.0` and Avo Inspector SDK version >= `0.1.0`.
+
+
+> `node-avo-inspector` dependency in your package.json file should be equal or greater than 0.1.0.
+
+> Generated Avo file exposes **avoInspectorApiKey** constant that contains your Inspector API key.
+
+This is how your Avo initialization code should look like:
+
+```javascript
+const Inspector = require("node-avo-inspector");
+
+const inspector = new Inspector.AvoInspector({
+  apiKey: Avo.avoInspectorApiKey,
+  env: env.DEV,
+  version: "1.0.0",
+  appName: "My App"
+});
+Avo.initAvo( { env: env.DEV, inspector: inspector}, ...);
```

Comment:
Should this say Avo.AvoEnv.Dev like the typescript code? Should we add the Avo import to be explicit?

Are the Typescript and Javascript examples the same? Would it be enough to have just one of them?

Reply:
Good catch, this actually should be a string in JS